src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyCacheableRequest.java

```diff
@@ -550,9 +584,9 @@ private void send503RetryAfter()
         factory.writer.doHttpResponse(acceptReply,
                                       acceptRouteId,
                                       acceptReplyId,
-                                      factory.supplyTrace.getAsLong(), e ->
-                                                        e.item(h -> h.name(STATUS).value(SERVICE_UNAVAILABLE_503))
-                                                         .item(h -> h.name("retry-after").value("0")));
+                                      factory.supplyTrace.getAsLong(),
+                                      e -> e.item(h -> h.name(STATUS).value(SERVICE_UNAVAILABLE_503))
+                                            .item(h -> h.name("retry-after").value("0")));
```

Comment:
Please convert these constant strings to constant `StringFW` and use here.

Reply:
Done.